README.md

```diff
@@ -1 +1,50 @@
-# Kanban-project
+## MOVIES
+This is a web application based on the TV Maze API. The API privides data about movies. We build an app about movies. It has 2 pages. The homepage and the comment page. On the homepage is a list of movies. You can like and comment on it.
+
+## Built With
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Jest
+
+
+## Live Demo
+You can view the live demo here
```

Comment:
- [ ] Kindly deploy the project and include the live deploy link here.

Reply:
Resolved

Reply:
- [ ]  [ optional] This live demo link seems to not be working properly (it is currently showing a white page), would you address that?
![screencapture-isabellanamutenda-github-io-Kanban-project-2021-09-17-20_22_02](https://user-images.githubusercontent.com/56886064/133836011-3ebc43f0-06af-4ca8-b6de-2acb1f8114cb.png)